api/fetch/index.js

```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const db = '../../dbConfig';
+
+router.get('', async (req, res) => {
+  try {
+    const { issueId } = req.body;
```

Comment:
Will this work when this isn't a post? This route should probably something like GET `/api/issues/:id/images`, since `/api/fetch` is so general